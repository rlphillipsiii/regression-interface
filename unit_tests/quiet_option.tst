#!/bin/sh

cd $REGRESSION_INT

name="quiet option"
flags="-q -r"

source unit_tests/common/setup
source unit_tests/common/dotest
