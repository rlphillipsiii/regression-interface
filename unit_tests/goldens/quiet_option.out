Dummy failing test
Dummy failing test on strict only
Dummy passing test
Running Test: Example Failing Test
Executing Command: ./setup
Executing Command: ./test
Test "Example Failing Test" failed.
Reason: The "DOTEST" command exited with a non-zero return code.

Running Test: Example Failing Test Setup
Executing Command: ./setup
[Errno 2] No such file or directory
Test "Example Failing Test Setup" failed.
Reason: Abnormal exit from "SETUP".

Running Test: Example Failing Test Strict
Executing Command: ./setup
Executing Command: ./test
Executing Command: ./cleanup

Running Test: Example Passing Test
Executing Command: ./setup
Executing Command: ./test
Executing Command: ./cleanup

